test/consoleagent.js

```diff
@@ -66,7 +66,7 @@ describe('ConsoleAgent', function () {
         let async = true;
         let compiled = agent.compile(program, {async});
 
-        assert.equal(compiled, `${ConsoleAgent.runtime}${program}`.replace(/\r?\n/g, ''));
+        assert.equal(compiled, `const name = 'ConsoleAgent';${program}`.replace(/\r?\n/g, ''));
```

Comment:
@rwaldron I took a stab at fixing this test, but I'm not thrilled with hard-coding this.  I could probably add a comment stripping regex here or try to use the AST parser from the code.